packages/app/src/components/choropleth/hooks/use-tab-interactive-button.tsx

```diff
@@ -0,0 +1,73 @@
+import css from '@styled-system/css';
+import { useCallback, useMemo, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { useHotkey } from '~/utils/hotkey/use-hotkey';
+
+export function useTabInteractiveButton(label: string) {
+  const [isTabInteractive, setIsTabInteractive] = useState(false);
+  const toggleButtonRef = useRef<HTMLButtonElement>(null);
+  const timeoutRef = useRef<number>();
+
+  useHotkey('escape', () => toggleButtonRef.current?.focus(), {
+    disabled: !isTabInteractive,
```

Comment:
```suggestion
    isDisabled: !isTabInteractive,
```

nit; Since it's not a prop I think this is more correct